js/index.js

```diff
@@ -1,7 +1,63 @@
-// Iteration 1: Names and Input
+//Iteration 1: Names and Input
+const hacker1 = 'karlish';
+console.log("The driver's name is " + hacker1);
 
+const hacker2 = 'karlish';
+console.log("The navigator's name is " + hacker2);
 
-// Iteration 2: Conditionals
+//Iteration 2: Conditionals
+if(hacker1.length > hacker2.length){
+  console.log('The driver has the longest name, it has ' + hacker1.length + ' characters.');
+} else if(hacker1.length < hacker2.length) {
+   console.log('It seems that the navigator has the longest name, it has ' + hacker2.length + ' characters.');
+} else {
+  console.log('Wow, you both have equally long names, ' + hacker1.length + ' characters!');
+}
 
+//Iteration 3: Loops
+let driver = '';
+for(let i = 0; i < hacker1.length; i++){
+  driver = driver+hacker1[i].toUpperCase()+' ';
+}
+console.log(driver);
+
+console.log(hacker2.split('').reverse().join(''));
```

Comment:
Esto esta bien pero solo estas imprimiento el resultado, otra manera de solucionarlo para asignarlo a algun lugar como una variable seria de esta manera:
` let navigator = hacker2.split('').reverse().join('')`

Aunque la mejor practica es hacerlo en una funcion, una forma de hacerlo seria asi:
`let str2 = "";`
`for(let i = hacker2.length-1; i >= 0; i--){`
  `str2 += `${hacker2[i]}``
`}`